src/screens/TopicPageScreen/index.js

```diff
@@ -1,36 +1,36 @@
-import * as React from 'react';
+import {useNavigation, useRoute} from '@react-navigation/native';
 import _ from 'lodash';
-import SimpleToast from 'react-native-simple-toast';
+import * as React from 'react';
 import {Animated, Platform, View} from 'react-native';
-import {useNavigation, useRoute} from '@react-navigation/native';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import SimpleToast from 'react-native-simple-toast';
 
 import BlockComponent from '../../components/BlockComponent';
-import BottomSheetFollow from './elements/BottomSheetFollow';
 import ButtonAddPostTopic from '../../components/Button/ButtonAddPostTopic';
-import MemoizedListComponent from './MemoizedListComponent';
-import NavHeader from './elements/NavHeader';
-import ShareUtils from '../../utils/share';
 import TiktokScroll from '../../components/TiktokScroll';
-import TopicPageStorage from '../../utils/storage/custom/topicPageStorage';
-import dimen from '../../utils/dimen';
-import removePrefixTopic from '../../utils/topics/removePrefixTopic';
-import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
-import useFeedPreloadHook from '../FeedScreen/hooks/useFeedPreloadHook';
-import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
+import {Context} from '../../context';
+import {setFeedByIndex, setTopicFeedByIndex, setTopicFeeds} from '../../context/actions/feeds';
 import useOnBottomNavigationTabPressHook, {
   LIST_VIEW_TYPE
 } from '../../hooks/navigation/useOnBottomNavigationTabPressHook';
-import {COLORS} from '../../utils/theme';
-import {Context} from '../../context';
-import {downVote, upVote} from '../../service/vote';
 import {getFeedDetail} from '../../service/post';
 import {getTopicPages} from '../../service/topicPages';
-import {getTopics, getUserTopic} from '../../service/topics';
-import {getUserId} from '../../utils/users';
+import {getTopics} from '../../service/topics';
+import {downVote, upVote} from '../../service/vote';
+import dimen from '../../utils/dimen';
+import {normalize} from '../../utils/fonts';
 import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
-import {normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
-import {setFeedByIndex, setTopicFeedByIndex, setTopicFeeds} from '../../context/actions/feeds';
+import ShareUtils from '../../utils/share';
+import TopicPageStorage from '../../utils/storage/custom/topicPageStorage';
+import {COLORS} from '../../utils/theme';
+import removePrefixTopic from '../../utils/topics/removePrefixTopic';
+import {getUserId} from '../../utils/users';
+import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+import useFeedPreloadHook from '../FeedScreen/hooks/useFeedPreloadHook';
+import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
+import MemoizedListComponent from './MemoizedListComponent';
+import BottomSheetFollow from './elements/BottomSheetFollow';
+import NavHeader from './elements/NavHeader';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made in this hunk are primarily about reordering the imports. While this doesn't affect the functionality of the code, it's a good practice to keep related imports together for better readability and maintainability. However, ensure that these changes do not introduce any circular dependencies or break any existing import order dependencies.
```diff
+import {useNavigation, useRoute} from '@react-navigation/native';
 import _ from 'lodash';
-import * as React from 'react';
+import * as React from 'react';
 import {Animated, Platform, View} from 'react-native';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
 import SimpleToast from 'react-native-simple-toast';
 
 import BlockComponent from '../../components/BlockComponent';
 import ButtonAddPostTopic from '../../components/Button/ButtonAddPostTopic';
 import TiktokScroll from '../../components/TiktokScroll';
 import {Context} from '../../context';
 import {setFeedByIndex, setTopicFeedByIndex, setTopicFeeds} from '../../context/actions/feeds';
 import useOnBottomNavigationTabPressHook, {
   LIST_VIEW_TYPE
 } from '../../hooks/navigation/useOnBottomNavigationTabPressHook';
 import {getFeedDetail} from '../../service/post';
 import {getTopicPages} from '../../service/topicPages';
 import {getTopics} from '../../service/topics';
 import {downVote, upVote} from '../../service/vote';
 import dimen from '../../utils/dimen';
 import {normalize} from '../../utils/fonts';
 import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
 import ShareUtils from '../../utils/share';
 import TopicPageStorage from '../../utils/storage/custom/topicPageStorage';
 import {COLORS} from '../../utils/theme';
 import removePrefixTopic from '../../utils/topics/removePrefixTopic';
 import {getUserId} from '../../utils/users';
 import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
 import useFeedPreloadHook from '../FeedScreen/hooks/useFeedPreloadHook';
 import useViewPostTimeHook from '../FeedScreen/hooks/useViewPostTimeHook';
 import MemoizedListComponent from './MemoizedListComponent';
+import BottomSheetFollow from './elements/BottomSheetFollow';
 import NavHeader from './elements/NavHeader';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->